cockroachcloud/serverless-faqs.md

```diff
@@ -105,6 +105,14 @@ You can submit feedback or log any bugs you find through [this survey](https://f
 
 Yes, we use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.3.
 
+### What certificates do I need to connect to my cluster?
+
+All connections to {{ site.data.products.serverless }} require SSL encryption. When connecting to your cluster using the CockroachDB SQL client or many drivers and ORMs, you don't need to download a root certificate and configure your client to use that certificate because the client will connect using the system root certificates. If you configure your client to use SSL and to verify the certificates (for example, by setting `sslmode=verify-full` in your [connection string](../{{site.versions["stable"]}}/connection-parameters.html#additional-connection-parameters)), your connection will be encrypted.
+
+Some drivers and ORMs don't use the system root certificates, however. In those cases, you need to download a root certificate file and configure your client to use that certificate when connecting to your cluster. You can [download the certificate](connect-to-a-serverless-cluster.html#step-2-connect-to-your-cluster) by following the instructions in the {{ site.data.products.db }} Console. Configure your client to use this certificate (for example, by setting `sslrootcert=<path to the root certificate>` in your connection string) and to use SSL (for example, by setting `sslmode=verify-full` in your connection string) to connect to your cluster.
```

Comment:
```suggestion
However, some drivers and ORMs don't use the system root certificates. In those cases, you need to download a root certificate file and configure your client to use that certificate when connecting to your cluster. You can [download the certificate](connect-to-a-serverless-cluster.html#step-2-connect-to-your-cluster) by following the instructions in the {{ site.data.products.db }} Console. Configure your client to use this certificate (for example, by setting `sslrootcert=<path to the root certificate>` in your connection string) and to use SSL (for example, by setting `sslmode=verify-full` in your connection string) to connect to your cluster.
```

Reply:
Are there other ways to configure the client?

Reply:
There's no standard way, so it depends on the driver/ORM. Many use the libpq connection string format that we favor, but others (JDBC and C# prominently) use different connection strings or have individual parameters.